specification/cluster/allocation_explain/ClusterAllocationExplainRequest.ts

```diff
@@ -44,6 +45,22 @@ export interface Request extends RequestBase {
     }
   ]
   query_parameters: {
+    /**
+     * Specifies the node ID or the name of the node to only explain a shard that is currently located on the specified node.
+     */
+    current_node?: string
```

Comment:
we prefer using aliases rather than primitive types when possible
```suggestion
    current_node?: NodeId
```